app/Http/Requests/ChangeStatusRequest.php

```diff
@@ -25,7 +25,8 @@ public function authorize()
     public function rules()
     {
         return [
-            'status' => 'required|integer|in:'.implode(',', [Order::PENDING, Order::ACCEPTED, Order::REJECTED])
+            'status' => 'required|integer|in:'.implode(',', [Order::PENDING, Order::ACCEPTED, Order::REJECTED]),
```

Comment:
'status' => 'required|integer|in:'.implode(',', [Order::PENDING, Order::ACCEPTED, Order::REJECTED]),
nó khác gì với
'status' => 'required|integer|in:'.Order::PENDING.','. Order::ACCEPTED.','. Order::REJECTED,

Reply:
dòng đó em thêm dấu phẩy thôi anh

Reply:
vì sao em lại dùng hàm implode?